articles/migrations/guides/extensibility-node8.md

```diff
@@ -116,17 +126,24 @@ to
 
 `var mysql = require(‘mysql’);`
 
+
+or, if the module must be pinned to a specific version:
+
+`var mysql = require(‘mysql@2.0.0’);`
+
 ### Behavioral and syntactic changes
 
-Some of the behavioral and syntactic changes in modules were not forward-compatible with Node.js v8.
+Some of the behavioral and syntactic changes in modules were not forward-compatible with Node 8.
 
 For example, the default encoding of the **crypto** module was changed from **binary** to **utf8**, and the used of **new Buffer()** has been deprecated in favor of **Buffer.from()**.
```

Comment:
`and the used of` sounds like the `d` in used shouldn't be there. Also, looks like `new Buffer()` and `Buffer.from()` should use code formatting instead of bold.